script/jenkins-utilities

```diff
@@ -151,7 +151,7 @@ function create_service() {
     || error_exit 'Could not fetch the service version.'
 
   echo 'Activating service'
-  retry -n 3 run ${GCLOUD} service-management enable \
+  retry -n 10 run ${GCLOUD} service-management enable \
```

Comment:
gcloud service-management deploy command will activate the service.
I think we can remove this.

I saw a case this command was failed. So I increased the number of retry.